src/utils/validateEmail.js

```diff
@@ -0,0 +1,13 @@
+const validateEmail = (email) => {
+  if (!email) {
+    return 'email is required';
+  }
+
+  const emailPattern = /^[\w.+-]+@([\w-]+\.){1,3}[\w-]{2,}$/;
+
+  if (!emailPattern.test(email)) {
+    return 'email is not valid';
+  }
+};
```

Comment:
Consider returning a default value (such as null or undefined) when the email is valid. This will make it clear that the function always returns a value, which can be useful for consistency in error handling.